addon/components/perfect-scroll/component.js

```diff
@@ -44,18 +44,57 @@ export default Ember.Component.extend({
   useKeyboard: true,
   suppressScrollX: false,
   suppressScrollY: false,
-  scrollTop: 0,
-  scrollLeft: 0,
   scrollXMarginOffset: 0,
   scrollYMarginOffset: 0,
   includePadding: false,
   theme: 'default',
 
+  _scrollTop: 0,
+  scrollTop: computed('_scrollTop', {
+    get() {
+      return get(this, '_scrollTop') || 0;
+    },
+    set(key, value) {
+      set(this, `_${key}`, value);
+      run.schedule('afterRender', () => {
+        get(this, '_scrollElement')[key] = value;
+        window.Ps.update(get(this, '_scrollElement'));
+      });
+    }
+  }),
+  _scrollLeft: 0,
+  scrollLeft: computed('_scrollLeft', {
+    get() {
+      return get(this, '_scrollLeft') || 0;
+    },
+    set(key, value) {
+      set(this, `_${key}`, value);
+      run.schedule('afterRender', () => {
+        get(this, '_scrollElement')[key] = value;
+        window.Ps.update(get(this, '_scrollElement'));
+      });
+    }
+  }),
+
+  scrolled(evt){
+    this.setProperties({
```

Comment:
Not sure about throttling or debouncing, might reduce the usefulness of the bindings. Maybe if the wait period was short enough, but not sure how much would be gained then.

I think the main cost of listening to these events is the listeners/handlers themselves, not the properties we set in the handler. 

Thoughts?